src/main/java/com/kat/bachaat/controller/UserController.java

```diff
@@ -13,11 +13,22 @@
 @RestController
 @RequestMapping("/user")
 public class UserController {
-    private static Logger logger = Logger.getLogger(UserController.class.getName());
+
+    Logger logger = Logger.getLogger(UserController.class.getName());
 
     @Autowired
     private UserService userService;
 
+    @DeleteMapping("/delete")
+    public ResponseEntity<Boolean> deleteUser(@RequestParam long id) {
+        logger.info("fetch deleteUser method");
+        boolean user = userService.deleteUser(id);
```

Comment:
what happens if id from parameter is null?

Reply:
done